app/templates/components/to-review-project-card.hbs

```diff
@@ -43,37 +43,38 @@
       </div>
     </div>
     <div class="cell medium-8 large-auto">
-
-      {{#if assignment.hearingsNotSubmittedNotWaived}}
-        <div class="grid-x">
-          <div class="cell medium-auto">
-            <LinkTo
-              @route="my-projects.assignment.hearing.add"
-              @model={{assignment}}
-              class="button expanded tiny-margin-bottom"
-              data-test-button-post-hearing="{{assignment.id}}"
-            >
-              {{fa-icon 'calendar' fixedWidth=true size="lg" transform="left-2"}}
-              Post {{participant-type-label assignment.dcpLupteammemberrole}} Hearing Notice
-            </LinkTo>
-          </div>
-          <div class="cell medium-shrink">
-            <button
-              class="button expanded gray"
-              onClick={{action "openOptOutHearingPopup"}}
-              data-test-button="optOutHearingOpenPopup"
-            >
-              {{fa-icon 'calendar-times' size="lg" transform="shrink-5 down-1"}}
-              <small>Opt Out</small>
-            </button>
+      {{#if this.assignment.dispositionsByRole}}
```

Comment:
Instead of not showing the hearings button at all, let's make it disabled (like how the recommendation button is conditionally disabled).

Reply:
@andycochran sure. this will need to be applied to the upcoming tab though.

Reply:
I'm going to open this in another PR this needs to get merged now.